includes/manage_pages.inc

```diff
@@ -568,27 +568,27 @@ function islandora_paged_content_ingest_zipped_pages(AbstractObject $object, $al
  *   Drupal form definition.
  */
 function islandora_paged_content_zipped_upload_form(array $form, array &$form_state, $pid, $allowed_extensions, $model, $derivatives) {
-  module_load_include('inc', 'islandora_ocr', 'includes/utilities');
   module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
   $object = islandora_object_load($pid);
   $current_pages = islandora_paged_content_get_pages($object);
   $last_page_number = count($current_pages);
-  $languages = module_exists('islandora_ocr') ? islandora_ocr_get_enabled_tesseract_languages() : array();
   $upload_size = min((int) ini_get('post_max_size'), (int) ini_get('upload_max_filesize'));
   $extensions = array('zip');
   $form = array();
-  $do_ocr = in_array('ocr', $derivatives) && module_exists('islandora_ocr');
   $message = t("This sequence currently has @count pages. Additional pages will be appended to the end of the sequence by default. !break", array("@count" => $last_page_number, '!break' => '<br />'));
   $message .= t("Choose a number lower than @count to insert page(s) at a specific location in the sequence.", array("@count" => $last_page_number, '!break' => '<br />'));
 
-  $form['language'] = array(
-    '#access' => module_exists('islandora_ocr') && $do_ocr,
-    '#title' => t('Language'),
-    '#type' => 'select',
-    '#description' => t('Please select the language the page is written in.'),
-    '#options' => $languages,
-    '#default_value' => 'English',
-  );
+  if (module_exists('islandora_ocr') && in_array('ocr', $derivatives)) {
+    module_load_include('inc', 'islandora_ocr', 'includes/utilities');
+    $form['language'] = array(
+      '#title' => t('Language'),
+      '#type' => 'select',
+      '#description' => t('Please select the language the page is written in.'),
+      '#options' => islandora_ocr_get_enabled_tesseract_languages(),
+      '#default_value' => 'English',
```

Comment:
I know this was existing code, but should we be assuming that "English" is installed? Or if it isn't will the system just select the first language?

Reply:
... incidentally, I guess this really wouldn't have ever worked, since the `#default_value` would actually have to be `eng`, no? So, will fix this and also make it make sense